### PR TITLE
[new] Add choices.js v2.0.2

### PR DESCRIPTION
**Pull request by iv-craig**
First time I'm adding a package, so I'm not sure if I've got the auto-update/git commit checklists right. Please let me know if there's anything I need to update.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.
# Profile of the lib
- Git repository (required): https://github.com/jshjohnson/Choices
- Official website (optional, not the repository): https://joshuajohnson.co.uk/Choices/
- NPM package url (optional): https://www.npmjs.com/package/choices.js
- GitHub / Bitbucket popularity (required):
  - Count of stars: 369
  - Count of watchers: 14
  - Count of forks: 22
- NPM download stats (optional):
  - Downloads in the last day: 214
  - Downloads in the last week: 865
  - Downloads in the last month: 2599
# Essential checklist
- [ ] I'm the author of this library
  - [ ] I would like to add link to the page of this library on CDNJS on website / readme
- [x] This lib was not found on cdnjs repo
- [x] No already exist / duplicated issue and PR
- [x] The lib has notable popularity
  - [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
  - [x] More than 500 downloads stats per month on npm registry
- [x] Project has public repository on famous online hosting platform (or been hosted on npm)
# Auto-update checklist
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct.
# Git commit checklist
- [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
- [x] The parent of the commit(s) in the PR is not old than 3 days.
- [x] Pull request is sending from a non-master branch with meaningful name.
- [ ] Separate unrelated changes into different commits.
- [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
- [ ] Close corresponding issue in commit message
- [ ] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
@cdnjs/intern2 please help review this PR, thanks.

File: ajax/libs/choices.js/package.json [hunk 1]
```diff
@@ -0,0 +1,76 @@
+{
+  "name": "choices.js",
+  "version": "2.0.2",
+  "description": "A vanilla JS customisable text input/select box plugin",
+  "main": "./assets/scripts/dist/choices.min.js",
```
Comment (pvnr0082t) [inline ajax/libs/choices.js/package.json]: @iv-craig Let's remove some fields we don't need. Like `main`, `scripts`, `bugs`, `devDependencies`.

File: ajax/libs/choices.js/package.json [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/jshjohnson/Choices.git"
+  },
+  "author": "Josh Johnson",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/jshjohnson/Choices/issues"
+  },
+  "homepage": "https://github.com/jshjohnson/Choices#readme",
```
Comment (pvnr0082t) [inline ajax/libs/choices.js/package.json]: We can remove `homepage` field because the homepage point to Github repo. We'll use `repository` info first.

**Comment by kennynaoh** [comment]
Hello @iv-craig ,
Please add `filename` filed, like `"filename":"choices.min.js"`
Please remove `main`, `scripts`, `bugs`, devDependencies` field.
Thanks!

**Comment by kennynaoh** [comment]
Hello @iv-craig 
Maybe you can change the `npmFileMap`.
{
     "basePath": "assets/scripts/dist",
     "files": [
          "**/*"
      ]
},
{
     "basePath": "assets/styles",
     "files": [
           "css/+(base|choice)*"
     ]
},
{
       "basePath": "assets/icons",
       "files": [
            "*.svg"
      ]
}
Thanks!

**Comment by iv-craig** [comment]
Hi @kennynaoh
I ran the package.json through the fixFormat.js tool, but CircleCI is still showing an error with the formatting. Does anything look off there? Thanks!

**Comment by PeterDaveHello** [comment]
@iv-craig Did you run under the root of cdnjs by ./tools/fixFormat.js?

**Comment by iv-craig** [comment]
@PeterDaveHello I had put the fixFormat functionality into a gulpfile and ran that locally to avoid cloning the whole cdnjs repo, but even after cloning the repo and running it from the cdnjs root it doesn't appear to have made any changes.

**Comment by PeterDaveHello** [comment]
@iv-craig any error there? Can you try to run it via command line instead of via gulp? Thanks.

**Comment by PeterDaveHello** [comment]
@kennynaoh the filemap looks a little bit complex, could it be easier and cleaner? 
@cdnjs/intern2 can you give other advices? Thanks.

**Comment by pvnr0082t** [comment]
It can pass the test after I ran `tools/fixFormat.js`. But I diff the package.json before and after I ran `tools/fixFormat.js`, they are totally the same.

**Comment by iv-craig** [comment]
@PeterDaveHello @pvnr0082t Committing from the command-line instead of the browser seems to have worked -- doing a sparse checkout also helped, I was getting errors during an earlier clone attempt. Thanks!

**Comment by PeterDaveHello** [comment]
@iv-craig Congratulations!

**Comment by PeterDaveHello** [comment]
@cdnjs/intern2 any updates here?

**Comment by x09326** [comment]
@iv-craig 
Please add ```keywords``` field in ```package.json```.

**Comment by PeterDaveHello** [comment]
@kennynaoh the filemap looks a little bit complex, could it be easier and cleaner?

**Comment by x09326** [comment]
@PeterDaveHello 
How about this?

{
     "basePath": "assets",
     "files": [
          "scripts/dist/**/*",
          "styles/css/+(base|choice)*",
          "icons/*.svg"
      ]
},

**Comment by PeterDaveHello** [comment]
We may need to slow down this PR, the structure looks not so good and common, can somebody help discuss with the author about the possibility to distribute all the distribution purpose files under a single `dist`?  Thanks.

**Comment by PeterDaveHello** [comment]
Any updates here? Thanks!

**Comment by iv-craig** [comment]
No updates, sorry. Been under a time crunch for...a while, and it's not letting up any time soon. Feel free to close this if you'd like.

**Comment by PeterDaveHello** [comment]
@iv-craig no worries, I'll take care of it.

**Pull request closed without merging**